src/app/site/cart/components/summary/summary.component.ts

```diff
@@ -89,10 +89,14 @@ export class SummaryComponent implements OnInit {
   pay() {
     this.cartService.applyCoupon(this.coupon_code)
       .then(res => {
-        this.router.navigate(['checkout']);
+        this.router.navigate(['/checkout']);
```

Comment:
Is it work?
When adding "/" before the url, it consider current url and then navigate (In other words it concatenate the current url and specified url)